components/button/index.css

```diff
@@ -89,6 +100,15 @@ governing permissions and limitations under the License.
   --spectrum-button-top-to-text: var(--spectrum-button-top-to-text-extra-large);
   --spectrum-button-bottom-to-text: var(--spectrum-button-bottom-to-text-extra-large);
 
+  .spectrum--medium &[pending],
+  .spectrum--medium &.is-pending {
+    --mod-button-edge-to-visual-only: calc(3px + var(--spectrum-button-edge-to-visual-only));
+  }
+
+  .spectrum--large &[pending],
+  .spectrum--large &.is-pending {
+    --mod-button-edge-to-visual-only: calc(4px + var(--spectrum-button-edge-to-visual-only));
+  }
```

Comment:
I couldn't find a token value that properly reflected the distance from button edge to the progress circle since it has different dimensions than the icons, so this is pretty custom. Should these be custom tokens? Or maybe there is a token value somewhere that I missed (I was [looking here](https://github.com/adobe/spectrum-tokens/blob/29663e130dc9f221462de87ae7df4847e325cb4b/packages/tokens/src/layout.json#L596))?